pallets/parachain-staking/src/weights.rs

```diff
@@ -435,14 +418,14 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: ParachainStaking DelayedPayouts (r:0 w:1)
 	#[rustfmt::skip]
 	fn round_transition_on_initialize(x: u32, y: u32, ) -> Weight {
-		(0 as Weight)
-			// Standard Error: 827_000
-			.saturating_add((65_228_000 as Weight).saturating_mul(x as Weight))
+		(105_685_000 as Weight)
+			// Standard Error: 819_000
+			.saturating_add((52_388_000 as Weight).saturating_mul(x as Weight))
 			// Standard Error: 2_000
```

Comment:
Again, does this look legit or just noise? This moves a lot of per-x and per-y scaling into a constant base fee.

Reply:
I presume this is related to reading the new storage item `DelegationScheduledRequests`. Unlike other functions this is the actual function that we wanted to minimize. Given that the `DelegationScheduledRequests` is constant time, it makes sense that it was added to base weight.